Add example starting from .mzML file and vendor file format

Y0dler opened the pull request:
Closes #32.
The mzML file was too large to be uploaded to GitHub. A different solution will have to be found.

michaelosthege commented:
The notebook looks good 👍

Please add it to the documentation index too.

You can upload the mzML file as an asset to the latest release:

https://github.com/JuBiotech/peak-performance/releases/edit/v0.7.1

Y0dler commented:
The notebook was modified from an older one so it's already in the index :)  
  
The idea with attaching the `*.mzML` file to the release was really good so that's what I just did.
I updated the instructions in the notebook accordingly.  
  
Also, I realized that two cell outputs of the notebook were extremely long and were shown in their entirety in the documentation so I changed that while I was at it.
